R/parse-block.R

```diff
@@ -126,9 +130,23 @@ parseBlock <- function(lineNum, file){
       if (!is.na(s) && !s %in% names(.globals$serializers)){
         stop("No such @serializer registered: ", s)
       }
+      shortSerAttr <- trimws(shortSerMat[1,3])
+      if(!identical(shortSerAttr, "") && !grepl("^\\(.*\\)$", shortSerAttr)){
```

Comment:
It would be good to have tests for edge cases so that we can more safely refactor in the future.